Frameworks/QuartzCore/CALayer.mm

```diff
@@ -2569,21 +2569,38 @@ inline id _createRtProxy(Class cls, IInspectable* iface) {
 + (CGPoint)convertPoint:(CGPoint)point fromLayer:(CALayer*)fromLayer toLayer:(CALayer*)toLayer {
     CGPoint ret = { point.x, point.y };
 
-    if (fromLayer && toLayer) {
-        // get the backing xaml UIElement for fromLayer
-        WXUIElement* fromLayerElement = _getBackingXamlElementForCALayer(fromLayer);
+    if (!fromLayer && !toLayer) {
+        return ret;
+    }
 
-        // get the backing xaml UIElement for toLayer
-        WXUIElement* toLayerElement = _getBackingXamlElementForCALayer(toLayer);
+    if (!fromLayer) {
+        // get the top most window for current view
+        fromLayer = toLayer;
+        while (fromLayer.superlayer != nil) {
+            fromLayer = fromLayer.superlayer;
+        }
+    } 
+    
+    if (!toLayer) {
+        toLayer = fromLayer;
+        while (toLayer.superlayer != nil) {
+            toLayer = toLayer.superlayer;
+        }
+    }
```

Comment:
i can revert the change here and wait for your change to come in. but this is going to leave corresponding issue hanging around a little longer given ux/uxe will probably take longer to arrive develop. On the other hand, for the merge conflicts that is going to occur if I leave this change here, you can just reject mine and accept yours when merging since you already have equivalent?  Let me know what you think. I can do either.

---
In reply to: [86420400](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86420400) [](ancestors = 86420400)